aws/resource_aws_waf_byte_match_set.go

```diff
@@ -117,8 +117,8 @@ func resourceAwsWafByteMatchSetUpdate(d *schema.ResourceData, meta interface{})
 
 	log.Printf("[INFO] Updating ByteMatchSet: %s", d.Get("name").(string))
 
-	if d.HasChange("byte_match_tuples") {
```

Comment:
The existing logic for `byte_match_tuples` must be preserved in addition to the new `byte_match_tuple`